ts/TextField/index.tsx

```diff
@@ -1,21 +1,54 @@
 import {Component, h} from 'preact';
 
-import Icon from '../Icon';
 import {MDCTextField} from '@material/textfield';
+import Icon from '../../Icon/index';
+import {
+  MDCComponent,
+  MDCFoundation,
+  MDCRipple
+} from '../../MaterialComponentsWeb';
 import MaterialComponent from '../MaterialComponent';
 
+declare class MDCTextFieldFoundation extends MDCFoundation<MDCTextField> {
```

Comment:
why is there a necessity to define this entire foundation and textfield class?
this is a huge addition to maintain this. can we work without declaring these classes?

Reply:
They where just copied over from the original `.d.ts`, I didn't write them, I'll look, if some one did this for us (under `@types`)